### PR TITLE
Side panel with text representation

### PR DESCRIPTION
**Pull request by akaDuality**
Check #14 for more details.

The issue isn't fixed at all, but we should merge this changes.

### CODE REVIEW

File: VoiceOver Designer/Features/Sources/TextUI/Array+Move.swift [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import Document
+
+extension Array where Element == A11yDescription {
```
Comment (drucelweisse) [inline VoiceOver Designer/Features/Sources/TextUI/Array+Move.swift:3]: is https://developer.apple.com/documentation/Swift/MutableCollection/move(fromOffsets:toOffset:) would be helpful?

**Review by drucelweisse — APPROVED** [review]
Looks dope, a lot of changes!

File: VoiceOver Designer/Features/Sources/TextUI/Array+Move.swift (hunk 1 above)
Comment (akaDuality) [reply VoiceOver Designer/Features/Sources/TextUI/Array+Move.swift:3]: Thank you, will check. Strange, but I haven't found this it autocomplete

Comment (akaDuality) [reply VoiceOver Designer/Features/Sources/TextUI/Array+Move.swift:3]: I can't call it :-(

Comment (drucelweisse) [reply VoiceOver Designer/Features/Sources/TextUI/Array+Move.swift:3]: Looking after time, it seems that function is defined in SwiftUI Extension for MutableCollection. I've asked that on [Swift Forums](https://forums.swift.org/t/mutablecollection-move-fromoffsets-cannot-be-called-without-importing-swiftui/59483)

**Pull request merged**